### ISSUE TITLE
dem_mosaic blending could be smoother

### ISSUE THREAD

**Issue by oleg-alexandrov** [NeoGeographyToolkit/StereoPipeline#143]
In some situations, the dem_mosaic weights do not decay smoothly at some portions of boundary. An example was added to the nightly test suite illustrating this: ss_dem_mosaic_unsmooth_wt

**Comment by oleg-alexandrov** [comment]
I studied this in detail. This is in the nature of the grassfire weights. Where the boundaries of two DEMs intersect, their grassfire weights are equally small, and there is no good way of transitioning between the two, resulting in a blending artifact sometimes.

Better weights are what Ara used to use, not grassfire, but based on how far the boundary is horizontally and vertically. But these are unable to handle holes well. 

For now I added to dem_mosaic the option --use-centerline-weights that uses Ara's idea. These seem to work nicer for blending DEMs which are tiles, rarther than having irregular boundaries. 

There should be a way of combing grassfire with Ara's weights. Food for thought.
